lib/dor/services/client.rb

```diff
@@ -127,6 +127,25 @@ def create_release_tag(object:, release:, what:, to:, who:)
           release_tags.create(object: object, release: release, what: what, to: to, who: who)
         end
 
+        # Open new version for an object
+        # @param object [String] object identifier
+        # @param params [Hash] optional params (see dor-services-app)
+        # @raise [UnexpectedResponse] when the response is not successful.
+        # @raise [MalformedResponse] when the response is not parseable.
+        # @return [String] the current version
+        def open_new_version(object:, **params)
```

Comment:
I wonder what you think of not having this method and just having the consumer do `client.object(druid).new_version(params`?

Reply:
I like the idea of hiding internal client complexity from consumers, and making all methods callable from the main Client class. (At some point, this list of methods may grow cumbersome, sure.)

Reply:
@jcoyne 	If you're cool with this, mind having another look?